ebsco/edsapi-koha-plugin
# Updates to allow custom links to work in guest access

Allows custom links to work in 20.11 when not logged in.

To Test:
http://koha-2011.ebsco-gss.net/
Search "Shakespeare" in the Discovery tab.
Click a PDF Link and Linked Full Text link to verify it prompts to log in.
Limit the Content Provider to DOAJ (Directory of Open Access Journals)
Click a Custom Link for DOAJ.  Note that you are given access without logging in.
